frontend/i18next.scanner/fr/translation.json

```diff
@@ -424,6 +424,8 @@
   "Password:": "Mot de passe :",
   "Favorite": "Favori",
   "My favorites": "Mes favoris",
+  "Change into content": "Transformer en contenu",
+  "Error while changing content type": "Une erreur est survenue lors de la transformation en contenu",
```

Comment:
I would prefer "Turn into content" in English.

Chat Gpt said about that:
```text
    "Convert to Content": This phrase has a slightly technical or formal tone. It's clear and direct, making it suitable for professional or instructional contexts.

    "Turn into Content": This phrase has a more casual or conversational tone. It's straightforward and easy to understand, making it suitable for user-friendly interfaces or informal instructions.

    "Change into Content": This phrase also has a formal tone, but it may imply a broader sense of modification rather than a specific action. It could be suitable for contexts where the transformation process is more complex or involves multiple steps.
```

Also, these 2 new translation keys should be in each languages translation files

Reply:
I use Turn into Content